pkg/kubectl/describe.go

```diff
@@ -382,13 +382,13 @@ func (d *ResourceQuotaDescriber) Describe(namespace, name string, describerSetti
 func helpTextForResourceQuotaScope(scope api.ResourceQuotaScope) string {
 	switch scope {
 	case api.ResourceQuotaScopeTerminating:
-		return "Matches all pods that have an active deadline."
+		return "Affects pods that have an active deadline. These pods usually include builds, deployers, and jobs."
```

Comment:
I don't think is appropriate for kubernetes.  The concept of builds and deployers don't exist in kube and `Affects` seems like the wrong verb.  The old description read accurately to me and this one does not.